README.md

```diff
@@ -28,9 +28,59 @@ import { NgxSherlockModule } from '@politie/ngx-sherlock';
 export class AppModule { }
 ```
 
+### `autoDetectChanges`
+
+**Signature**:
+```typescript
+autoDetectChanges(detectorRef: ChangeDetectorRef): void;
+```
+
+The function `autoDetectChanges` will automatically run a change detection cycle when `Derivables` or `DerivableProxies` within a component's template change internal state. The function should be called in an `OnInit` lifecycle hook of a component or directive.
+
+The `autoDetectChanges` function guarantees model and view fidelity, meaning one can easily use Angular's forms and template functionality.
+
+#### Example
+
+`trusty-sidekick.component.html`:
+```html
+<h2>Well there you are, {{sidekick$.firstname.$value}} {{sidekick$.surname.$value}}!</h2>
+<input type="text" [(ngModel)]="sidekick$.firstname.$value" placeholder="First name">
+<input type="text" [(ngModel)]="sidekick$.surname.$value" placeholder="Surname">
+```
+
+`trusty-sidekick.component.ts`:
+```typescript
+import { Component, OnInit, ChangeDetectionStrategy, ChangeDetectorRef } from '@angular/core';
+import { autoDetectChanges } from '@politie/ngx-sherlock';
+import { atom } from '@politie/sherlock';
+import { ProxyDescriptor } from '@politie/sherlock-proxy';
+
+@Component({
+    selector: 'trusty-sidekick',
+    templateUrl: 'trusty-sidekick.component.html',
+    changeDetection: ChangeDetectionStrategy.onPush,
+})
+export class TrustySidekickComponent implements OnInit {
+
+    private readonly sidekickAtom$ = atom({ firstname: 'John', surname: 'Watson' });
```

Comment:
I would prefer to leave out the "Atom" part in reactive variable names. The `$` already conveys that.